Update link for the Printable Markdown Cheatsheet

james-alt opened the pull request:
It looks like cheatsheet was removed with commit https://github.com/MicrosoftDocs/azure-docs/commit/337eb2f45e2671d1bde43bdb4719b874d50f0b63 so updating the reference to the official GitHub markdown cheatsheet to match the markdown basics.

azurecla commented:
Hi __@shontnew__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@shontnew @jeffgoll 
I'll merge this PR when I see a comment here to do so. Thanks.

jeffgoll commented:
#sign-off

james-alt committed:
Update link for the Printable Markdown Cheatsheet

It looks like cheatsheet was removed with commit https://github.com/MicrosoftDocs/azure-docs/commit/337eb2f45e2671d1bde43bdb4719b874d50f0b63 so updating the reference to the official GitHub markdown cheatsheet to match the markdown basics.
